shell/security/prismaci/twist-scan.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+# This script scans docker images for vulnerabilities, generates local
+# report and also uploads the report to Prisma Cloud. The script expects
+# special credentials - those are set by prismacloud-credentials circleci context.
+#
+
+## Add some safeties for external dependencies when running
+if [ -z "${CIRCLECI}" ]; then
+  echo "I can only run in CircleCI"
+  exit 1
+fi
+if ! [ -x "$(which curl)" ]; then 
+  echo "We need an executable called curl in the $PATH in order to execute"
+  exit 1
+fi
+if ! [ -x "$(which jq)" ]; then 
+  echo "We need an executable called curl in the $PATH in order to execute"
+  exit 1
+fi
```

Comment:
style: prefer built-ins to external programs
```suggestion
if ! command -v curl >/dev/null 2>&1; then 
  echo "We need an executable called curl in the $PATH in order to execute"
  exit 1
fi
if ! command -v jq >/dev/null 2>&1; then 
  echo "We need an executable called curl in the $PATH in order to execute"
  exit 1
fi
```